controller/notify.go

```diff
@@ -1,18 +1,30 @@
 package controller
 
 import (
+	"context"
 	"fmt"
 
 	"github.com/fabric8-services/fabric8-notification/app"
 	"github.com/fabric8-services/fabric8-notification/collector"
 	"github.com/fabric8-services/fabric8-notification/email"
 	"github.com/fabric8-services/fabric8-notification/jsonapi"
 	"github.com/fabric8-services/fabric8-notification/template"
+	"github.com/fabric8-services/fabric8-notification/token"
 	"github.com/fabric8-services/fabric8-wit/errors"
 	"github.com/fabric8-services/fabric8-wit/log"
 	"github.com/goadesign/goa"
 )
 
+const (
+	CVENotification             string = "analytics.notify.cve"
+	UserEmailUpdateNotification string = "user.email.update"
+)
+
+var nTypeToSAMap = map[string]string{
```

Comment:
Could we get this into the Registry or somewhere instead of a separate thing?

Reply:
@aslakknutsen let me re-factor this and overall notification_type stuff.